src/firecracker/examples/uffd/uffd_utils.rs

```diff
@@ -292,3 +292,92 @@ fn create_mem_regions(mappings: &Vec<GuestRegionUffdMapping>) -> Vec<MemRegion>
 
     mem_regions
 }
+
+#[cfg(test)]
```

Comment:
I think you also need to do something like 

```diff
diff --git a/tests/integration_tests/build/test_unittests.py b/tests/integration_tests/build/test_unittests.py
index 2919fb90a..312235715 100644
--- a/tests/integration_tests/build/test_unittests.py
+++ b/tests/integration_tests/build/test_unittests.py
@@ -23,6 +23,7 @@ def test_unittests(test_fc_session_root_path):
 
     extra_args = f"--target {TARGET}"
     host.cargo_test(test_fc_session_root_path, extra_args=extra_args)
+    host.cargo_test(test_fc_session_root_path, extra_args=extra_args + "--examples")

```

to get the CI to run the tests

Reply:
added